api/v1beta1/metallb_types.go

```diff
@@ -88,6 +88,14 @@ type MetalLBSpec struct {
 	// The type of BGP implementation deployed with MetalLB
 	// +optional
 	BGPBackend params.BGPType `json:"bgpBackend,omitempty"`
+
+	// The specific frrk8s configuration
+	FRRK8SConfig *FRRK8SConfig `json:"frrk8sConfig,omitempty"`
+}
+
+type FRRK8SConfig struct {
+	// A comma separated list of cidrs we want always to block for incoming routes
+	AlwaysBlock string `json:"alwaysBlock,omitempty"`
```

Comment:
shouldn't we have it as a list here and make it comma separated as part of the logic? imo it feels cleaner from a k8s api perspective

Reply:
yep, I guess you are right